### ISSUE TITLE
Possible to change switch values?

### ISSUE THREAD

**Issue by rbarman** [cgarvis/angular-toggle-switch#76]
Currently it seems that the possible switch values are true and false. 
Is it possible to add our own values that are more appropriate to our application?

			<toggle-switch
    			ng-model="ctrl.model.side"
    			on-label="For"
                on-value = "For"
    			off-label="Against">
                of-value = "Against"
			<toggle-switch>

So in this case, the value for the 'On' case is "For" rather than true. 
Ideally it would be nice to modify the default values for our own applications. Is there a way to do this?

**Comment by cgarvis** [comment]
If it's something you would use so much that a global config makes sense for you, why not wrap this into your own directive?  

```html
<for-against-toggle-switch ng-model="ctrl.model.side" />
```

**Comment by zachlysobey** [comment]
I too think this would be a very useful enhancement. Personally I don't know if my angular chops are good enough that I could successfully wrap in my own directive without a `$watch` function for each instance.

**Comment by sean-hill** [comment]
I agree, I'm converting my checkboxes in my app to this toggle and on one of them I have `ng-true-value="false" ng-false-value="true"` for a specific case.

**Comment by sean-hill** [comment]
Just made a pull request [here](https://github.com/cgarvis/angular-toggle-switch/pull/88).
